app-web/Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM node:8
```

Comment:
Here is some stuff I often add to my Dockerfiles:
```
ARG NODE_MAJOR_VERSION=10
ARG NODE_VERSION=10.14.x
ARG APP_VERSION="0.1.0"

ENV INSTALL_PATH /opt/app
ENV SUMMARY="MDMS Device Provisioning API ${APP_VERSION}"  \
  DESCRIPTION="MDMS Device Provisioning API {APP_VERSION} running node nodejs ${NODE_VERSION} on Alpine Linux"
ENV LC_ALL=en_US.UTF-8 \
  LANG=en_US.UTF-8 

LABEL summary="$SUMMARY" \
  description="$DESCRIPTION"
```

Then I'd use $INSTALL_PATH as a variable in line like `WORKDIR $INSTALL_PATH`

Reply:
added those in, thanks!